.github/workflows/build.yml

```diff
@@ -21,11 +21,27 @@ jobs:
       with:
         fetch-depth: 0
         
+    - name: .NET info (before setup)
+      run: |
+        dotnet --info
+        printenv PATH
+        printenv MSBuildSDKsPath
```

Comment:
I don't think there is an MSBuildSDKsPath before you execute the setup-dotnet github action - same way not sure the dotnet commands would work either